integrations/amazonKeys/cli/init.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash -e
+readonly IFS=$'\n\t'
+readonly ROOT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+readonly SCRIPT_NAME="$( basename "$0" )"
+readonly ARGS="$@"
+
+export RESOURCE_NAME=""
+export SCOPE=""
+
+print_help() {
+  echo "
+  Usage:
+    $ROOT_DIR/$SCRIPT_NAME <resource_name> [scope]
```

Comment:
- doesnt need root_dir. its a command and can be placed anywhere.
- it should also support `--help` parameter.